GoCardlessSDK/GoCardlessSDK/Classes/Network/HttpClient.swift

```diff
@@ -32,12 +32,14 @@ class HttpClient {
             let encoded = try? JSONEncoder().encode(body)
             request.httpBody = encoded
         }
+        print(request)
```

Comment:
do you actually want to print this? there are a few other `prints` in this PR

Reply:
Oh thanks, I missed a couple. Fixed them